dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/bean/TelemetryBeanTestServlet.java

```diff
@@ -76,13 +77,15 @@ public void testBaggageUpdates() {
 
         Baggage newBaggage = Baggage.builder().put("foo", "bar").build();
         logger.info("Baggage created during the test: " + newBaggage.toString());
-        newBaggage.makeCurrent();
-
-        Baggage baggageTwo = Baggage.current();
-        logger.info("Baggage.current() after creating a new baggage and making it current : " + baggageTwo.asMap());
+        try (Scope s = newBaggage.makeCurrent()) {
+            Baggage baggageTwo = Baggage.current();
+            logger.info("Baggage.current() after creating a new baggage and making it current : " + baggageTwo.asMap());
+
+            assertEquals("Didn't find expected value in injected baggage", "bar", injectedBaggage.getEntryValue("foo"));
+            assertEquals("The current baggage was not the one we created and invoked makeCurrent() with", newBaggage.asMap(), baggageTwo.asMap());
+            assertEquals("The injected baggage was not the same as the current baggage after calling makeCurrent()", baggageTwo.asMap(), injectedBaggage.asMap());
+            s.close();
```

Comment:
Because you're in a try-with-resources block, you don't need to close scope here, it's closed automatically when you leave the block.